lib/tests/streamlit/runtime/caching/hashing_test.py

```diff
@@ -391,6 +413,72 @@ def test_generator_not_hashable(self):
         with self.assertRaises(UnhashableTypeError):
             get_hash((x for x in range(1)))
 
+    def test_hash_funcs_acceptable_keys(self):
```

Comment:
This could also use a docstring, because the purpose of the test isn't very clear from its name. (Maybe the comment in the middle of the function is sufficient?)